app/experimenter/nimbus-ui/src/components/PageHome/DirectoryTable/index.tsx

```diff
@@ -365,11 +368,11 @@ const DirectoryTable: React.FunctionComponent<DirectoryTableProps> = ({
               </tr>
             ))}
           </tbody>
-        </table>
+        </Table>
       ) : (
         <p data-testid="no-experiments">No experiments found.</p>
       )}
-    </div>
+    </>
```

Comment:
Same as above, can this be removed?

Reply:
it is required to get rid of the additional wrapping div